demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -120,6 +120,25 @@ def normalize_name(self) -> str:
         # if has name attribute, return it, otherwise return the object id
         return self.object_id
 
+    def __add_metadata(self, json_dct: Dict):
+        """
+        Add extra metadata to the model.
+
+        Args:
+            json_dct: dict representation of the object
+
+        """
+        if "path" in json_dct and Path(json_dct["path"]).is_absolute():
+            json_dct["path"] = (Path(json_dct["path"]).relative_to(CONTENT_PATH)).as_posix()  # type: ignore
+
+        json_dct["content_type"] = self.content_type
+
+        if self.content_type in (ContentType.SCRIPT, ContentType.INTEGRATION):
+            python_version = self.get_python_version()  # type: ignore[attr-defined]
```

Comment:
maybe we can override IntegrationScript's to_dict() instead?

Reply:
implemented it differently